pkgs/applications/misc/koreader/default.nix

```diff
@@ -63,7 +66,7 @@ stdenv.mkDerivation rec {
     description =
       "An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats, running on Cervantes, Kindle, Kobo, PocketBook and Android devices";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    platforms = intersectLists platforms.x86_64 platforms.linux;
+    platforms = with platforms; intersectLists linux (x86_64 ++ aarch64);
```

Comment:
```suggestion
    platforms = [ "aarch64-linux" "x86_64-linux" ];
```

Reply:
No idea what the general guidance is here but happy with either format.

Reply:
not sure why it was originally using `intersectLists`, writing them out literally just made more sense to me